sources/packages/backend/apps/db-migrations/src/sql/StudentAssessments/Add-calculation-start-date.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+    sims.student_assessments
+ADD
+    COLUMN calculation_start_date TIMESTAMP WITH TIME ZONE;
```

Comment:
Should we add some script to have the current records on DB updated?
For instance, copy the `assessment_date`, when present, to the new `calculation_start_date` or set now() when the status is already `in progress`?

Reply:
i believe this is only a problem for test and dev IMO, as @dheepak-aot said we can leave things as it is or clear the assessment data in dev and test after this implementation.